code_search/README.md

```diff
@@ -0,0 +1,52 @@
+# Semantic Code Search
+
+End-to-End Semantic Code Search on Kubeflow
+
+## Prerequisites
+
+* Python 2.7 (with `pip`)
```

Comment:
Hey @jlewi  the example I open sourced was only tested with python `3.6` just heads up

Reply:
Noted. I'd also like to point out that the data ingestion from BigQuery and further processing via Apache Beam is an isolated step (and can be run independently in a Python2 environment). I am planning to do the model training and serving steps in `Python3.6+` as well.